packages/solid-crs-components/lib/forms/form-element.component.ts

```diff
@@ -391,6 +399,9 @@ export class FormElementComponent<T> extends RxLitElement {
         :root {
           display: block;
         }
+        .icon {
+          pointer-events: none;
+        }
```

Comment:
@lem-onade some technical explanation: `pointer-evenets: none` will make an object not clickable (or any other events ofc). This will act like you are clicking on the element behind it.

Reply:
Cool. Are both `element.pointerEvents = 'none'` and `.icon { pointer-events: none; }` needed?

Reply:
the one in the css block is for the initial load of the page.
The one on the `focusout` event will reset the `pointer-events: 'initial'` set by opening the dropdown. Because when the dropdown is open I assume you want to be able to click the arrow to close it again. Setting this is `initial` will emit that `focusout` event when the arrow is clicked.